pkg/controller/beat/common/stackmon/stackmon.go

```diff
@@ -103,6 +103,19 @@ func MetricBeat(ctx context.Context, client k8s.Client, beat *v1beta1.Beat) (sta
 		},
 	})
 
+	// Add shared volume for logs consumption.
+	sidecar.Container.VolumeMounts = append(sidecar.Container.VolumeMounts, corev1.VolumeMount{
+		Name:      "metricbeat-logs",
```

Comment:
`const` ?

Reply:
Any reason not use  `volume.NewEmptyDirVolume("metricbeat-logs", "/usr/share/metricbeat/logs")`

Reply:
I added a constant to `beat/common/stackmon` and updated all controllers to use this for consistency, and updated all to use NewEmptyDirVolume.